src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Attr.java

```diff
@@ -5606,6 +5588,9 @@ private void attribClassBody(Env<AttrContext> env, ClassSymbol c) {
             }
         }
 
+        // Check for proper placement of super()/init() calls.
```

Comment:
```suggestion
        // Check for proper placement of super()/this() calls.
```

Reply:
Thanks for catching that - fixed in 0e638da2daa.